### PR TITLE
sistecredito icon

### PR DESCRIPTION
**Pull request by gomonke**
## Why are you adding this icons?
I'm adding/updating this icon(s) because we have the application approved and listed at https://apps.shopify.com/sistecredito-app. We need to update the payment method icon

#### Help us identify yourself
- [x] I'm working/collaborating with the brand directly and they have provided the icons.
- [ ] I'm associated with the brand and I've read all the brand icon’s guidelines.
- [ ] I'm an individual and I've read all the brand icon’s guidelines.

Link to the brand guidelines: <Link>

## Checklist to add new icons
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I have attached a screenshot comparison with the example icon provided in guidelines
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

## Attach a screenshot of the icon along side the example Visa icon
icon-compare

#### Tips how to create a screenshot
We have found free online SVG editor https://www.freecodeformat.com/svg-editor.php very useful to create one. Here is a sample code for you to verify that you icon appears properly along side the placeholder.

```
<!-- Change background color if needed to showcase your icon better -->
<style> body { background: black; } </style>

<!-- DO NOT DELETE EXAMPLE -->
<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" role="img" width="38" height="24" aria-labelledby="pi-visa"><title id="pi-visa">Visa</title><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path d="M28.3 10.1H28c-.4 1-.7 1.5-1 3h1.9c-.3-1.5-.3-2.2-.6-3zm2.9 5.9h-1.7c-.1 0-.1 0-.2-.1l-.2-.9-.1-.2h-2.4c-.1 0-.2 0-.2.2l-.3.9c0 .1-.1.1-.1.1h-2.1l.2-.5L27 8.7c0-.5.3-.7.8-.7h1.5c.1 0 .2 0 .2.2l1.4 6.5c.1.4.2.7.2 1.1.1.1.1.1.1.2zm-13.4-.3l.4-1.8c.1 0 .2.1.2.1.7.3 1.4.5 2.1.4.2 0 .5-.1.7-.2.5-.2.5-.7.1-1.1-.2-.2-.5-.3-.8-.5-.4-.2-.8-.4-1.1-.7-1.2-1-.8-2.4-.1-3.1.6-.4.9-.8 1.7-.8 1.2 0 2.5 0 3.1.2h.1c-.1.6-.2 1.1-.4 1.7-.5-.2-1-.4-1.5-.4-.3 0-.6 0-.9.1-.2 0-.3.1-.4.2-.2.2-.2.5 0 .7l.5.4c.4.2.8.4 1.1.6.5.3 1 .8 1.1 1.4.2.9-.1 1.7-.9 2.3-.5.4-.7.6-1.4.6-1.4 0-2.5.1-3.4-.2-.1.2-.1.2-.2.1zm-3.5.3c.1-.7.1-.7.2-1 .5-2.2 1-4.5 1.4-6.7.1-.2.1-.3.3-.3H18c-.2 1.2-.4 2.1-.7 3.2-.3 1.5-.6 3-1 4.5 0 .2-.1.2-.3.2M5 8.2c0-.1.2-.2.3-.2h3.4c.5 0 .9.3 1 .8l.9 4.4c0 .1 0 .1.1.2 0-.1.1-.1.1-.1l2.1-5.1c-.1-.1 0-.2.1-.2h2.1c0 .1 0 .1-.1.2l-3.1 7.3c-.1.2-.1.3-.2.4-.1.1-.3 0-.5 0H9.7c-.1 0-.2 0-.2-.2L7.9 9.5c-.2-.2-.5-.5-.9-.6-.6-.3-1.7-.5-1.9-.5L5 8.2z" fill="#142688"/></svg>

<!-- TODO: insert your icon here -->
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-sistecredito"><title id="pi-sistecredito">SISTECREDITO</title><defs><style>.cls-1,.cls-2{fill:#3ec92b;stroke-width:0}.cls-2{fill:#00369c}</style></defs><path class="cls-2" d="M11.6 11.6v-.9h-.9v-.9H10c0 .3 0 .4-.1.5-.1.2-.4.4-.6.4v.8h.4V13c0 .3 0 .6.3.8.2.2.5.3.9.3h.8v-.8h-.2-.5s-.1-.2 0-.3v-1.3h.9zM1 13.7l.5-.7.6.3H3s.1-.1.1-.2v-.2h-1c-.3-.1-.6-.2-.8-.4-.2-.2-.3-.4-.3-.7 0-.3.1-.6.4-.8.3-.2.7-.3 1.1-.3.2 0 .5 0 .7.1.2 0 .5.2.7.3l-.5.7c-.2 0-.3-.2-.5-.2h-.8s-.1 0-.1.2v.2h1c.3.1.6.2.8.4.2.2.3.4.3.7 0 .2 0 .3-.1.5 0 .1-.2.3-.3.4-.1.1-.3.2-.5.2h-.6c-.3 0-.6 0-.9-.1-.3 0-.5-.2-.7-.3zM5.8 13.7l.5-.7.6.3h.9s.1-.1.1-.2v-.2h-1c-.3-.1-.6-.2-.8-.4-.2-.2-.3-.4-.3-.7 0-.3.1-.6.4-.8.3-.2.7-.3 1.1-.3.2 0 .5 0 .7.1.2 0 .5.2.7.3l-.5.7c-.2 0-.3-.2-.5-.2h-.8s-.1 0-.1.2v.2h1c.3.1.6.2.8.4.2.2.3.4.3.7 0 .2 0 .3-.1.5 0 .1-.2.3-.3.4-.1.1-.3.2-.5.2h-.6c-.3 0-.6 0-.9-.1-.3 0-.5-.2-.7-.3z"/><path class="cls-2" d="M15.1 13.7c-.2.2-.4.3-.7.4-.2 0-.5.1-.8.1-.2 0-.5 0-.7-.1-.2 0-.4-.2-.6-.4-.2-.2-.3-.3-.4-.6 0-.2-.1-.4-.1-.7 0-.2 0-.5.1-.7 0-.2.2-.4.4-.6.2-.2.3-.3.6-.4.2 0 .5-.1.7-.1.5 0 .9.2 1.2.5.2.2.3.4.4.6 0 .2.1.5.1.7v.3h-2.4c0 .2.2.3.3.4.2.1.3.2.5.1h.4c.1 0 .2-.1.3-.2l.7.6zm-1.5-2.2c-.2 0-.3 0-.4.1-.1 0-.2.2-.3.4h1.4c0-.2-.1-.3-.3-.4-.1 0-.3-.1-.4-.1zM4.4 9.8c0 .3.3.6.6.5.3 0 .5-.2.5-.5H4.4zM4.4 10.7h1v3.4h-1v-3.4z"/><g><path class="cls-1" d="M33.1 11.6v-.9h-.9v-.9h-.7c0 .3 0 .4-.1.5-.1.2-.4.4-.6.4v.8h.4V13c0 .3 0 .6.3.8.2.2.5.3.9.3h.8v-.8H33h-.5s-.1-.2 0-.3v-1.3h.9zM17.4 13.3h.4c.1 0 .2-.2.3-.3l.6.6c-.2.2-.4.4-.6.5-.2.1-.5.2-.8.2-.2 0-.5 0-.7-.1-.7-.3-1.1-.9-1.1-1.6 0-.2 0-.5.1-.7 0-.2.2-.4.4-.6.3-.3.8-.5 1.3-.5.3 0 .5 0 .8.2.2.1.5.3.6.5l-.6.7c-.1-.1-.2-.2-.4-.3h-.4c-.2 0-.4 0-.6.2-.2.2-.2.4-.2.6 0 .2 0 .4.2.6.1.2.4.2.6.2zM19 14.1v-3.4h1v.3c.1-.1.2-.2.4-.3.2 0 .3-.1.5-.1h.4v.9h-.2-.3c-.2 0-.3 0-.4.1-.1 0-.3.2-.3.3V14h-1zM22.7 9.8c0 .3.3.6.6.5.3 0 .5-.2.5-.5h-1.1zM24.8 13.7c-.2.2-.4.3-.7.4-.2 0-.5.1-.8.1-.2 0-.5 0-.7-.1-.4-.2-.8-.5-1-.9 0-.2-.1-.4-.1-.7 0-.2 0-.5.1-.7 0-.2.2-.4.4-.6.2-.2.3-.3.6-.4.2 0 .5-.1.7-.1.5 0 .9.2 1.2.5.2.2.3.4.4.6 0 .2.1.5.1.7v.3h-2.4c0 .2.2.3.3.4.2.1.3.2.5.1h.4c.1 0 .2-.1.3-.2l.7.6zm-1.5-2.2c-.2 0-.3 0-.4.1-.1 0-.2.2-.3.4H24c0-.2-.1-.3-.3-.4-.1 0-.3-.1-.4-.1zM27.9 14.1v-.2c-.1 0-.3.2-.4.2H27c-.2 0-.5 0-.7-.1-.2 0-.4-.2-.5-.4-.2-.2-.3-.3-.4-.6 0-.2-.1-.4-.1-.7 0-.2 0-.5.1-.7.2-.4.5-.7.9-.9.4-.1.8-.2 1.1 0 .1 0 .3 0 .4.2v-.8l1-.2v4.3h-1zm-1.6-1.7c0 .2 0 .5.3.6.2.2.4.3.6.2h.4c.1 0 .2-.1.3-.2v-1.2s-.2-.1-.3-.2h-.4c-.2 0-.5 0-.6.2-.2.2-.3.4-.3.6zM29.3 9.8c0 .3.3.6.6.5.3 0 .5-.2.5-.5h-1.1zM29.3 10.7h1v3.4h-1v-3.4z"/><path class="cls-1" d="M36.4 11.7h-.2l-.2.5v.2c0 .2 0 .4-.2.6-.3.3-.8.3-1.1 0-.3-.3-.3-.9 0-1.2.3-.3.8-.3 1.1 0 0 0 0 .1.1.2v-.3c.2-.2.4-.3.6-.2l.4.2c0-.2-.2-.4-.4-.5-.2-.2-.4-.3-.6-.4-.5-.2-1-.2-1.4 0-.2 0-.4.2-.6.4-.2.2-.3.3-.4.6-.2.4-.2.9 0 1.4 0 .2.2.4.4.6.2.2.4.3.6.4.5.2 1 .2 1.4 0 .2 0 .4-.2.6-.4.2-.2.3-.3.4-.6 0-.2.1-.4.1-.7V12l-.5-.2z"/></g></svg>

<br>
<!-- TODO: insert your icon here -->
<YOUR SVG CODE>
</br
```

#### If the icons are intended for use by Shopify, please provide the following info:
Who are you working with at Shopify? (avoid adding personal details, provide github handle(preferred) or first name and last name) @gomonke

What's the expected date of this change to deploy on Shopify? 04/15/2024

### CODE REVIEW

File: db/payment_icons.yml [hunk 1]
```diff
@@ -2206,3 +2206,7 @@
   name: orangemobilemoney
   label: Orange Mobile Money
   group: wallets
+-
+  name: sistecredito
+  label: SISTECREDITO
```
Comment (Lovedanihonjin) [inline db/payment_icons.yml:2211]: Is there a reason why uppercase is used here? If not, please change to sentence case.

File: app/assets/images/payment_icons/sistecredito.svg
Comment (Lovedanihonjin) [inline app/assets/images/payment_icons/sistecredito.svg:1]: SVG is missing background and border. Please refer to the guidelines

**Review by Lovedanihonjin — CHANGES_REQUESTED** [review]
Please refer to the icon guidelines. Design changes requested to the SVG file and .yml label.

**Review by Lovedanihonjin — APPROVED** [review]
Looks good. Thank you!

**Comment by gomonke** [comment]
Hello @dannye0231 , now that it has been approved for deployment, what should I do to have the icon available in the payment app extension settings?

**Pull request merged**